pubspec.yaml

```diff
@@ -4,42 +4,38 @@ description: A new Flutter project.
 # Prevent accidental publishing to pub.dev.
 publish_to: 'none'
 
-version: 1.0.1+2
+version: 1.1.0+1
 
 environment:
-  sdk: ">=3.4.0 <4.0.0"
+  sdk: ">=3.6.0 <4.0.0"
+  flutter: ">=3.27.1"
 
 dependencies:
-  auto_route: ^7.8.0
-  autoequal: ^0.5.1
-  copy_with_extension: ^5.0.0
-  equatable: ^2.0.3
+  auto_route: ^9.3.0+1
   flutter:
     sdk: flutter
-  flutter_bloc: ^8.0.1
+  flutter_bloc: ^9.0.0
+  flutter_lints: ^5.0.0
   flutter_localizations:
     sdk: flutter
-  hive: ^2.0.6
-  hive_flutter: ^1.1.0
-  sentry_flutter: ^7.9.0
-  freezed: ^2.4.1
-  freezed_annotation: ^2.2.0
-  json_annotation: ^4.8.0
+  freezed: ^2.5.7
```

Comment:
`freezed` should be located in the `dev_dependencies`, only `freezed_annotation` goes to runtime